c_plus_plus/src/model/audio/odas/odas_position_source.h

```diff
@@ -0,0 +1,44 @@
+#ifndef ODAS_POSITION_SOURCE_H
+#define ODAS_POSITION_SOURCE_H
+
+#include <memory>
+
+#include <QMutex>
```

Comment:
Here you could forward declare `QMutex` to make the header file lighter, in other words minimizing the chain of dependencies.